vcd/resource_vcd_vapp_vm.go

```diff
@@ -648,14 +651,17 @@ func resourceVcdVAppVmDelete(d *schema.ResourceData, meta interface{}) error {
 	log.Printf("[TRACE] vApp Status:: %s", status)
 	if status != "POWERED_OFF" {
 		log.Printf("[TRACE] Undeploying vApp: %s", vapp.VApp.Name)
-		task, err := vapp.Undeploy()
+		err = retryCall(vcdClient.MaxRetryTimeout, func() *resource.RetryError {
```

Comment:
Please add a comment on why this retry is needed.

Reply:
I added generic comment for all retryCall.